docs/section19/19-7.md

```diff
@@ -0,0 +1,84 @@
+# Section 19 - Rule 19-7               
+**Schema Version:** 0.0.24      
+**Mandatory Rule:** True    
+**Rule ID:** 19-7                 
+**Rule Description:**  Minimum ventilation system outdoor air intake flow shall be the same for the proposed design and baseline building design except when any of the 4 exceptions defined in Section G3.1.2.5 are met.  
+Exceptions included in this RDS:
+2. When designing systems in accordance with Standard 62.1, Section 6.2, “Ventilation Rate Procedure,” reduced ventilation airflow rates may be calculated for each HVAC zone in the proposed design with a zone air distribution effectiveness (Ez) > 1.0 as defined by Standard 62.1, Table 6-2. Baseline ventilation airflow rates in those zones shall be calcu-lated using the proposed design Ventilation Rate Procedure calculation with the following change only. Zone air distribution effectiveness shall be changed to (Ez) = 1.0 in each zone having a zone air distribution effectiveness (Ez) > 1.0. Proposed design and baseline build-ing design Ventilation Rate Procedure calculations, as described in Standard 62.1, shall be submitted to the rating authority to claim credit for this exception. M
+
```

Comment:
I think rule description should be uupdated to say that it also covers exception 1 (DCV) since in this case we exect min. OA rates in the baseline and proposed design to still be the same but the DCV control to be modeled.

Reply:
Leave it as is per call